pkgs/tools/security/wordlists/rockyou.nix

```
@@ -0,0 +1,31 @@
+{ fetchFromGitLab
+, lib
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rockyou";
+  version = "0.3-1kali3";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = "wordlists";
+    rev = "debian/${version}";
+    sha256 = "sha256-viOd8iKLTiHrPF0azICp/F16JrPUfo4kbcGUPD2/cRs=";
+  } + "/rockyou.txt.gz";
+
+  unpackCmd = ''
+    mkdir src
+    gzip -dc ${src} > src/rockyou.txt
```

we do a string interpolation anyway, so it might be a good idea to do this:
```suggestion
  };

  unpackCmd = ''
    mkdir src
    gzip -dc ${src}/rockyou.txt.gz > src/rockyou.txt
```

---

I prefer the original one: it makes it explicit that the source is only the `.txt.gz` file.